tools/p/p

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
nit: maybe containerized_p.sh or something?

Ah I see this is actually trying to allow callers to drop in here without installing p and run p directly.